.travis.yml

```diff
@@ -0,0 +1,14 @@
+dist: xenial
+language: python
+python:
+  - 3.7
+  - 3.8
+install:
+  - pip install -r requirements.txt
+  - pip install -r requirements-dev.txt
+script:
+  - coverage run -m pytest  # Run the tests and check for test coverage.
+  - coverage report -m  # Generate test coverage report.
+  - codecov  # Upload the report to codecov.
+  - flake8
+  - set -e  # Stop here if anything fails above
```

Comment:
This will cause anything that runs _below_ to stop the build. Since it is the last command, it has no effect and would be safe to remove. (It will be useful if we setup proper documentation and `doctr` later.)